src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -20,17 +20,17 @@ const { TYPES, ACTIONS } = require('../../manage_nsfs/manage_nsfs_constants');
 const tmp_fs_path = path.join(TMP_PATH, 'test_bucketspace_fs');
 const DEFAULT_FS_CONFIG = get_process_fs_context();
 const config_fs_account_options = { show_secrets: true, decrypt_secret_key: true };
+const root_path = path.join(tmp_fs_path, 'root_path_manage_nsfs/');
+const config_root = path.join(tmp_fs_path, 'config_root_manage_nsfs');
+const config_fs = new ConfigFS(config_root);
+// TODO: needed for NC_CORETEST FLOW - should be handled better
+const nc_coretes_location = config.NC_MASTER_KEYS_FILE_LOCATION;
```

Comment:
typo in the variable name, and I would add more details in the name.
```suggestion
const nc_coretest_master_key_location = config.NC_MASTER_KEYS_FILE_LOCATION;
```